File: ElasticsearchClient/src/app/shared/models/request/request-drop-down-values.ts
export class RequestDropDownValues {
  currentFilter: string;
  size: number;
  holidayYear: string[];
  regionName: string[];
  weekNumber: string[];
  responsibleRevenueManager: string[];
  parkName: string[];
  accommTypeName: string[];
  accommBeds: string[];
  unitGradeName: string[];
  accommName: string[];
  keyPeriodName: string[];

  constructor() {
    this.currentFilter = null;
    this.size = 1000;
    this.holidayYear = [null];
    this.regionName = [null];
    this.weekNumber = [null];
    this.responsibleRevenueManager = [null];
    this.parkName = [null];
    this.accommBeds = [null];
    this.accommName = [null];
    this.accommTypeName = [null];
    this.unitGradeName = [null];
    this.keyPeriodName = [null];
  }
}

File: BusinessLogic/Common/Models/Request/RequestDropDownValues.cs
using System.Collections.Generic;

namespace BusinessLogic.Common.Models.Request
{
    public class RequestDropDownValues
    {
        public string CurrentFilter { get; set; }
        public int Size { get; set; }
        public IEnumerable<string> HolidayYear { get; set; }
        public IEnumerable<string> RegionName { get; set; }
        public IEnumerable<string> WeekNumber { get; set; }
        public IEnumerable<string> ResponsibleRevenueManager { get; set; }
        public IEnumerable<string> ParkName { get; set; }
        public IEnumerable<string> AccommTypeName { get; set; }
        public IEnumerable<string> AccommBeds { get; set; }
        public IEnumerable<string> UnitGradeName { get; set; }
        public IEnumerable<string> AccommName { get; set; }
        public IEnumerable<string> KeyPeriodName { get; set; }
    }
}
